hilla/pom.xml

```diff
@@ -0,0 +1,173 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-platform-parent</artifactId>
+        <version>23.0-SNAPSHOT</version>
+    </parent>
+    <groupId>dev.hilla</groupId>
+    <artifactId>hilla</artifactId>
+    <packaging>jar</packaging>
+    <name>Hilla Platform</name>
+    <description>Hilla Platform</description>
+    <url>https://hilla.dev</url>
+
+    <distributionManagement>
+        <repository>
+            <id>vaadin-prereleases</id>
+            <url>https://maven.vaadin.com/vaadin-prereleases/</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>dev.hilla</groupId>
+                <artifactId>hilla-bom</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>fusion-endpoint</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-dev-server</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-client</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
We must remove lumo and material dependency here.
A pure TS project might want not to use vaadin-components nor vaadin-themes 

But previously we need to fix https://github.com/vaadin/flow/issues/12606

Reply:
hmm, I think it's fine to have those dependencies for now. As the main goal is to have a `hilla-bom` and dependency that a user can use.
Separate Hilla from Flow is the ultimate goal, but not necessarily to have it now. For example, `fusion-endpoint` depends on `flow-server` anyway.

Reply:
ok, let lave it for now